### DISCUSSION TITLE
[Feature] Rätsel Editor

### DISCUSSION THREAD

**Discussion by AMatutat** [Programmiermethoden/Dungeon#1378] [Ideas]
Über die DSL können verschiedene Aufgaben/Rätsel definiert und in das Spiel geladen werden.
Nachdem eine Aufgabe gelöst wurde, gibt ein, vorher in der DSL definiertes, Petri-Netz an, welches Level, also welche Aufgabe als nächstes geladen werden soll.

Es soll eine grafische Benutzeroberfläche zum erstellen von Rätseln und zum Einbetten dieser in ein Petri Netz konfektioniert und implementiert werden.

**Comment by cagix** [comment]
die dsl ist erstmal raus - aber man könnte hier eine art super-level-editor sehen ala unity und co. 

@AMatutat macht das sinn? dann wieder aufmachen.
